bcda/models/postgres/alr.go

```diff
@@ -0,0 +1,252 @@
+package postgres
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/gob"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/CMSgov/bcda-app/bcda/models"
+	"github.com/jackc/pgx"
+	"github.com/jackc/pgx/stdlib"
+	"github.com/sirupsen/logrus"
+)
+
+/*******************************************************************************
+	DATA STRUCTURES
+		1. AlrRepository
+		2. alrCopyFromSource
+			- Used to implement the CopyFrom from the pgx package CopyFrom is
+			Go's connection to PostgreSQL's COPY functionality.
+*******************************************************************************/
+
+type AlrRepository struct {
+	*sql.DB
+	*pgx.Conn
```

Comment:
Since we can retrieve the pgx.Conn from the db which is backed by a pool of connections, can we keep the *sql.Db reference and grab the pgx.Conn when we need it?